app/registration/models.py

```diff
@@ -6,11 +6,13 @@ class Registration(db.Model):
     id = db.Column(db.Integer(), primary_key = True)
     username = db.Column(db.Text(), unique = True)
     blob = db.Column(db.Text())
-    priority = db.Column(db.Integer(), unique = True)
+    priority = db.Column(db.Integer())
     confirmed = db.Column(db.Boolean())
     uni_id = db.Column(db.Integer(), db.ForeignKey('uni.id'))
     uni = db.relationship('Uni', backref=db.backref('Registrations', lazy='dynamic', cascade="all, delete-orphan"))
 
+    priority_constraint = db.UniqueConstraint('uni_id', 'priority', name='uni_priority_unique')
+
```

Comment:
After #24 is merged, I'll rebase this on master and generate a migration for the unique constraint here.